### PR TITLE
tooling, ci, cleanup

### PR DESCRIPTION
**Pull request by zeeshanlakhani**
Add similar checks/coverage/etc as we did w/ https://github.com/wnfs-wg/rs-wnfs

### CODE REVIEW

File: codecov.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+ignore:
+  - "*/test/*"
+  - "crates/wasm/*"
```
Comment (appcypher) [inline codecov.yaml:3]: Want to remove `crates/wasm/*` here.

File: codecov.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+comment:
+  layout: "reach, diff, flags, files"
+  require_changes: true
+
+github_checks:
+  annotations: false
+
+status:
```
Comment (appcypher) [inline codecov.yaml:12]: `status` field is unknown according to codecov.io/validate 

```bash
cat codecov.yaml | curl --data-binary @- https://codecov.io/validate
```

File: src/ratchet.rs [hunk 3]
```diff
@@ -1,11 +1,10 @@
-use std::fmt::{self, Display, Formatter};
-
 use crate::{
     constants::{LARGE_EPOCH_LENGTH, MEDIUM_EPOCH_LENGTH, RATCHET_SIGNIFIER},
     hash::Hash,
     PreviousErr, RatchetErr,
 };
 use rand::Rng;
+use std::fmt::{self, Display, Formatter};
```
Comment (appcypher) [inline src/ratchet.rs:7]: Is there a reason for choosing this order over having `std` stuff first?

File: codecov.yaml (hunk 1 above)
Comment (zeeshanlakhani) [reply codecov.yaml:3]: knew I missed one. Thanks.

File: src/ratchet.rs (hunk 3 above)
Comment (zeeshanlakhani) [reply src/ratchet.rs:7]: essentially, it's all abc order for everything, and most editors tend to format that way too.

File: codecov.yaml (hunk 2 above)
Comment (zeeshanlakhani) [reply codecov.yaml:12]: @appcypher derp, yep, i forgot coverage.

File: .github/CODEOWNERS [hunk 4]
```diff
@@ -0,0 +1,5 @@
+# Default
```
Comment (appcypher) [inline .github/CODEOWNERS:1]: Can you add `@ramfox`? They started the repo

Comment (zeeshanlakhani) [reply .github/CODEOWNERS:1]: yep. and will change our names to the team.

File: .github/CODEOWNERS [hunk 5]
```diff
@@ -0,0 +1,3 @@
...
+* @ramfox
+* @fission-codes/neutron
```
Comment (zeeshanlakhani) [inline .github/CODEOWNERS:3]: need to wait on some admin magic to make this validate.

**Review by appcypher — APPROVED** [review]
<div align="center">
    <a href="#" target="_blank">
        </img>
    </a>
</div>

<h3 align="center">LGTM! 🎉 </h3>

**Pull request merged**